ghost/i18n/locales/ru/comments.json

```diff
@@ -1,39 +1,39 @@
 {
-    "{{amount}} characters left": "{{amount}} символов осталось",
+    "{{amount}} characters left": "{{amount}} символа(ов) осталось",
     "{{amount}} comments": "{{amount}} комментария(ев)",
     "{{amount}} days ago": "{{amount}} дня(ей) назад",
     "{{amount}} hours ago": "{{amount}} часа(ов) назад",
     "{{amount}} minutes ago": "{{amount}} минут(ы) назад",
     "{{amount}} months ago": "{{amount}} месяца(ев) назад",
-    "{{amount}} more": "еще {{amount}}",
+    "{{amount}} more": "ещё {{amount}}",
     "{{amount}} seconds ago": "{{amount}} секунд(ы) назад",
     "{{amount}} weeks ago": "{{amount}} недель(и) назад",
     "{{amount}} years ago": "{{amount}} года(лет) назад",
     "1 comment": "1 комментарий",
     "Add comment": "Добавить комментарий",
-    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Добавьте подробности о себе, чтобы обсуждение стало более интерессным",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Дополните свой комментарий контекстом, добавив подробности о себе и своём опыте, чтобы обсуждение стало более интересным.",
     "Add reply": "Добавить ответ",
-    "Already a member?": "Уже присоединился?",
+    "Already a member?": "Уже присоединились?",
     "Anonymous": "Анонимный",
     "Become a member of {{publication}} to start commenting.": "Присоединяйтесь к {{publication}} чтобы начать комментировать.",
     "Become a paid member of {{publication}} to start commenting.": "Станьте платным участником {{publication}} чтобы начать комментировать.",
     "Cancel": "Отмена",
     "Comment": "Комментарий",
     "Complete your profile": "Заполните свой профиль",
     "Delete": "Удалить",
-    "Deleted member": "Удаленный участник",
+    "Deleted member": "Удалённый участник",
     "Discussion": "Обсуждение",
     "Edit": "Редактировать",
     "Edit this comment": "Редактировать этот комментарий",
     "Edited": "Отредактированно",
     "Enter your name": "Введите ваше имя",
     "Expertise": "Деятельность",
-    "Founder @ Acme Inc": "Основатель @ Acme Inc",
+    "Founder @ Acme Inc": "Основатель @ Корпорация «Акме»",
     "Full-time parent": "Родитель, работающий полный рабочий день",
-    "Head of Marketing at Acme, Inc": "Руководитель отдела маркетинга в Acme",
+    "Head of Marketing at Acme, Inc": "Руководитель отдела маркетинга в Корпорации «Акме»",
     "Hide": "Скрыть",
     "Hide comment": "Скрыть комментарий",
-    "Jamie Larson": "Вася Пупкин",
+    "Jamie Larson": "Павел Бид",
```

Comment:
Firstly: it was funny shorten form of a name and surname. Similar to: John Dick.

Secondly: According to the guideline here should be a typical unisex name. The thing is in Russian no unisex names because of lang rules. We have a few unisex names, but it is different when you write it (Alexander (male) and Alexandra (female)).